pyproject.toml

```diff
@@ -1,6 +1,44 @@
+[project]
+name = "epython"
+version = "2021.1002"
+description = "Helper library for QA"
+authors = [
+    "Ray Gomez <codenomad@gmail.com>"
+]
+license = "MIT"
+readme = "README.md"
+python = "^3.6"
+homepage = "https://github.com/elibs/epython"
+repository = "https://github.com/elibs/epython"
+documentation = "https://github.com/elibs/epython"
+
+keywords = ["packaging", "dependency", "infer", "pyproject.toml"]
+
+classifiers = [
+    "Topic :: Software Development"
+]
+
+
 [build-system]
 requires = [
     "setuptools>=42",
     "wheel"
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.bumpver]
+current_version = "2021.1002"
```

Comment:
Do you get automatic versioning with this? Nifty!

Reply:
Sort of, the idea is that you use the file patterns below to list the files that have version strings in it, so when you do need to do a version bump it's as easy as:

bumpver --update

That will commit and push the new version. This will be part of the github pipeline as soon as I can get it sorted ;)